dataset:
  name: Carla
  type: range_samples_from_image
  args:
    directory: ./dataset/Carla
    scene_ids:
      - Town01
      - Town02
      - Town03
      - Town04
      - Town05
      - Town06
    res_in: '16_1024'
    res_out: '128_2048'
    num_of_samples: 16384
    memory_fetch: false

model:
  name: liif_lidar
  output: ./models/trained/liif
  args:
    local_ensemble: true
    feat_unfold: false
    cell_decode: false